corehq/ex-submodules/casexml/apps/case/mock.py

```diff
@@ -239,10 +239,10 @@ def fmt(value):
                 return ''
             if isinstance(value, datetime):
                 return unicode(format_datetime(value))
-            elif isinstance(value, (basestring, int)):
+            elif isinstance(value, (basestring, int, date)):
```

Comment:
should we not call `json_format_date` similar to how `datetimes` are handled above?